src/sagemaker/workflow/step_collections.py

```diff
@@ -250,32 +254,32 @@ def __init__(
         self,
         name: str,
         estimator: EstimatorBase,
-        model_data,
-        model_inputs,
-        instance_count,
-        instance_type,
-        transform_inputs,
-        description: str = None,
-        display_name: str = None,
+        model_data: str,
+        model_inputs: CreateModelInput,
+        instance_count: int,
+        instance_type: str,
+        transform_inputs: TransformInput,
+        description: Optional[str] = None,
+        display_name: Optional[str] = None,
         # model arguments
```

Comment:
What does it mean by "model arguments"? Isn't model_data not model arguments?

Reply:
The arguments to construct the Model object.
https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/workflow/step_collections.py#L353